test/unit/spec/v2/view-builder/views/AuthenticatorIdpVerifyView_spec.js

```diff
@@ -0,0 +1,132 @@
+        authenticatorEnrollments,
+        app
+      }, {});
+
+      if (isFailure) {
+        appState.set('messages', {
+          'value': [
+            {
+              'message': 'Authentication failed. Please try again.',
+              'links': [
+                {'label': 'Help link 1', 'url': 'https://www.okta.com/'},
+                {'label': 'Help link 2', 'url': 'https://www.okta.com/help?page=1'}
+              ],
+              'i18n': {
+                'key': 'security.access_denied_custom_message'
+              },
+              'class': 'ERROR'
+            }
+          ]
+        });
+      }
+
+      jest.spyOn(appState, 'getRemediationAuthenticationOptions').mockReturnValue(formName => {
+        if (formName === 'select-authenticator-authenticate') {
+          return [ { label: 'some authenticator '} ];
+        }
+        return [];
+      });
+
+      const settings = new Settings({ baseUrl: 'http://localhost:3000', 'features.skipIdpFactorVerificationBtn': skipIdpFactorVerification });
+      const currentViewState = {
+        name: 'challenge-authenticator',
+        relatesTo: {
+          value: currentAuthenticator,
+        },
+      };
+      const ViewClass = isVerify ? AuthenticatorIdPVerifyView : AuthenticatorIdPEnrollView;
+      testContext.view = new ViewClass({
+        el: $sandbox,
+        appState,
+        settings,
+        currentViewState,
+      });
+      appStateTriggerSpy = jest.spyOn(testContext.view.options.appState, 'trigger');
+      testContext.view.render();
+    };
+  });
+
+  afterEach(function() {
+    $sandbox.empty();
+  });
+
+  it.each(['Verification', 'Enrollment']) ('shows idp authenticator %s screen with ' +
+        'SKIP_IDP_FACTOR_VERIFICATION_BUTTON disabled', function(requestName) {
+    testContext.init(requestName);
+    if (requestName === 'Verification') {
+      expect(testContext.view.$('.okta-form-subtitle').text()).toBe(
+        'You will be redirected to verify with IDP Authenticator'
+      );
+    } else {
+      expect(testContext.view.$('.okta-form-subtitle').text()).toBe(
+        'Clicking below will redirect to enrollment in IDP Authenticator'
+      );
+    }
+    expect(testContext.view.$('.o-form-button-bar').css('display')).toBe('block');
+    expect(testContext.view.$('.okta-waiting-spinner').css('display')).toBe('none');
+    expect(appStateTriggerSpy).toHaveBeenCalledTimes(0);
+  });
+
+  it.each(['Verification', 'Enrollment']) ('auto-submit idp authenticator %s screen with SKIP_IDP_FACTOR_VERIFICATION_BUTTON enabled', function(requestName) {
+
+    testContext.init(requestName, true);
+    if (requestName === 'Verification') {
+      expect(testContext.view.$('.okta-form-subtitle').text()).toBe(
+        'You will be redirected to verify with IDP Authenticator'
+      );
+    } else {
+      expect(testContext.view.$('.okta-form-subtitle').text()).toBe(
+        'Clicking below will redirect to enrollment in IDP Authenticator'
```

Comment:
Do we show this message even when we don't show the button? If we have auto-submit, we should probably change this text

Reply:
thanks @mauriciocastillosilva-okta. How about this for both cases autoredirect and non-autoredirect similar to verification? 

 `You will redirected to enroll with IDP Authenticator`